std/typecons.d

```diff
@@ -696,6 +746,12 @@ template Tuple(Specs...)
             assert(s[0] == "abc" && s[1] == 4.5);
         }
 
+        /**
+            Creates a hash of this `Tuple`.
+            
+            Returns:
+                A unique `size_t` representing the hash of this `Tuple`.
```

Comment:
IMO unqiue is wrong here. e.g. there can not be a unique hash of type size_t for Tuple!(size_t,size_t)